beast/physicsmodel/helpers/gridbackends.py

```diff
@@ -1,72 +1,85 @@
 """
 Backends to handle the model grids different ways
-
 =================================================
 
 Multiple backends are available to reduce the memory footprint for a
-performance cost as small as possible. They allow grids to be stored into FITS
-files but also into HDF5 format.
-
-In principle, HDFBackend is the most optimized as it allow you to attack the
-grid file directly without memory overhead thanks to the full use of pytables
-package.
+performance cost as small as possible.
 
 Implemented Backends
 --------------------
 
 MemoryBackend:
     Load everything into memory. Can initiate from variables, a filename,
-    CacheBackend, and HDFBackend.
+    CacheBackend, and DiskBackend.
 
 CacheBackend:
     Load data only at the first request. You can work using only seds or only
     model properties without the overhead of loading both. (works with FITS and
     HDF files). Offers also dropping part of the data.
 
-HDFBackend:
-    Works directly with an HDFStore support, ie., on disk. Cache and reading
-    are allowed through any way offered by pytables, which becomes very handy
+DiskBackend:
+    Works directly with an h5py support, ie., on disk. Cache and reading
+    are allowed through any way offered by h5py, which becomes very handy
     for very low-memory tasks such as doing single star figures.
+"""
+import sys
+import astropy.io.fits as fits
+import h5py
+import copy
+from astropy.table import Table
+from astropy.io.misc.hdf5 import read_table_hdf5
```

Comment:
Do we explicitly need hdpy and read_table_hdf5 in addition to astropy.table to read/write a HDF5 file?

Reply:
Yes.  The read_table_hdf5 allows reading a table from a hdu in a hdulit.  E.g., in memory w/o opening the file a 2nd time.  I couldn't figure out how to do this with the standard Table.read command.  There might be a way, but...